core/src/main/scala/stryker4s/config/implicits/ConfigReaderImplicits.scala

```diff
@@ -16,13 +19,26 @@ trait ConfigReaderImplicits {
     ConfigReader[Path].map(p => File(p))
 
   private[config] implicit val logLevelReader: ConfigReader[Level] = ConfigReader[String] map (
-    level => Level.valueOf(level))
+      level => Level.valueOf(level))
 
   private[config] implicit val toReporterList: ConfigReader[List[MutantRunReporter]] =
     ConfigReader[List[String]].map(_.map {
       case MutantRunReporter.`consoleReporter` => new ConsoleReporter
     })
 
   private[config] implicit val exclusions: ConfigReader[Exclusions] =
-    ConfigReader[List[String]].map(exclusions => Exclusions(exclusions.toSet))
+    ConfigReader[List[String]]
+      .map(warnInvalidExclusions)
+      .map(exclusions => Exclusions(exclusions.toSet))
+
+  private def warnInvalidExclusions(list: List[String]): List[String] = {
+    val (valid, invalid) = list.partition(Mutation.mutations.contains)
+    if (invalid.nonEmpty) {
+      val errorMessage =
+        s"""Invalid exclusion option(s): '${invalid.mkString(", ")}'
+            |Valid exclusions are ${Mutation.mutations.mkString(", ")}""".stripMargin
+      throw new Exception(errorMessage)
```

Comment:
Maybe a more specific exception would be clearer, and easier to catch if we ever want to catch it in the future (unlikely, but still). There already is a package `stryker4s.extensions.exceptions` where you could put it

Reply:
I think it should be an extension of `FailureReason` that's the pureconfig way of providing these errors.

[reference](https://pureconfig.github.io/docs/error-handling.html#custom-failure-reasons)

Reply:
@legopiraat I attempted this [here](https://github.com/stryker-mutator/stryker4s/pull/105/commits/4d84ee4c42692df7d929959b731b2c2457a3fa82#r235942059).